README.md

```diff
@@ -425,17 +421,23 @@ This message is used to communicate the current sonar state. If the data field i
 ### control
 
 #### 2600 reset
-Reset the sonar. The bootloader may run depending on the selection according to the `bootloader` payload field. When the bootloader runs, the external LED flashes at 5Hz. If the bootloader is not contacted within 3 seconds, it will run the current program. If there is no program, then the bootloader will wait forever for a connection.
+Reset the sonar. The bootloader may run depending on the selection according to the `bootloader` payload field. When the bootloader runs, the external LED flashes at 5Hz. If the bootloader is not contacted within 5 seconds, it will run the current program. If there is no program, then the bootloader will wait forever for a connection. Note that if you issue a reset then you will have to close all your open comm ports and go back to issuing either a discovery message for UDP or go through the break sequence for serial comms before you can talk to the sonar again.
 
 | Type | Name             | Description                                                      | Units |
 |------|------------------|------------------------------------------------------------------|-------|
-| u8 | bootloader | 0 = skip bootloader; 1 = run bootloader |  |
+| u16 | bootloader | 0 = skip bootloader; 1 = run bootloader |  |
```

Comment:
reserved can also be useful here, or we can use as a normal bitfield/value parameter. Maybe renaming from `bootloader` to `configuration` or `type` to be more future proof ?

Reply:
I don't believe there are any plans or any need for future capability here beyond bootloader and main program. For that reason, I would like to keep the bootloader name for explicit clarity.

I think the same reasoning can be applied to splitting this into two u8 fields (I believe is what you are suggesting): `bootloader` and `reserved`. I think this will only serve to obfuscate things. A single 16 bit field is of course overkill for a binary value, but it's a byproduct of the memory alignment, and it works.

Reply:
on second thought, I agree about reserved.